src/Form/FormProps.ts

```diff
@@ -4,13 +4,25 @@ import * as PropTypes from "prop-types";
 import { ModelInterface } from "../Model";
 import { FormContext } from "./FormContext";
 
+export interface LocalStorageRequiredInterface {
```

Comment:
Why not to use ES6 Storage interface?

Reply:
![image](https://user-images.githubusercontent.com/25794983/41456281-a6bf83d8-7088-11e8-9f14-914c061a163c.png)
Because ES6 Storage interface contains additional method, that not used in Form and not required